src/index.js

```diff
@@ -61,13 +61,14 @@ function rollem (configs, options) {
       if (err) {
         watcher.emit('failed', err)
       } else {
-        gazeWatcher.on('all', () => {
-          watcher.emit('changed')
-          buildBundles(configs).then(() => watcher.emit('rolled'))
+        gazeWatcher.on('all', (_event, filepath) => {
```

Comment:
Just out of curiosity: why do you need the underscore character for the event variable? Naming it simply "event" seems fine for me.